### DISCUSSION TITLE
I am getting a error i cant figure

### DISCUSSION THREAD

**Discussion by mikenld1** [WalletConnect/walletconnect-monorepo#1606] [Developer Support]
My dapp connects with web 3. Gets the adress, gets the contract adress. Than i get a error i cant figure out:

index.js?dd68:224 Uncaught (in promise) Error: bad response (status=400, headers={"content-length":"74","content-type":"application/json"}, body="{\"jsonrpc\":\"2.0\",\"error\":{\"code\":-32600,\"message\":\"invalid json request\"}}", requestBody="{\"method\":{\"jsonrpc\":\"2.0\",\"id\":3394823945030582,\"method\":\"eth_call\",\"params\":[{\"data\":\"0x13faede6\",\"to\":\"0xaa0c8a0ce725bf734b685498739c26ac2fd201e4\"},\"latest\"]},\"id\":42,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://rpc.walletconnect.com/v1/?chainId=eip155:137&projectId=a28dc568ea31bc4224916d00dfcf0190", code=SERVER_ERROR, version=web/5.7.1)
    at Logger.makeError (index.js?dd68:224:1)
    at Logger.throwError (index.js?dd68:233:1)
    at eval (index.js?8139:231:1)
    at Generator.next (<anonymous>)
    at fulfilled (index.js?8139:5:43)

and in the browser:

Unhandled Runtime Error
Error: bad response (status=400, headers={"content-length":"74","content-type":"application/json"}, body="{\"jsonrpc\":\"2.0\",\"error\":{\"code\":-32600,\"message\":\"invalid json request\"}}", requestBody="{\"method\":{\"jsonrpc\":\"2.0\",\"id\":3394823945030582,\"method\":\"eth_call\",\"params\":[{\"data\":\"0x13faede6\",\"to\":\"0xaa0c8a0ce725bf734b685498739c26ac2fd201e4\"},\"latest\"]},\"id\":42,\"jsonrpc\":\"2.0\"}", requestMethod="POST", [url="https://rpc.walletconnect.com/v1/?chainId=eip155:137&projectId=a28dc568ea31bc4224916d00dfcf0190",](http://localhost:3000/url=%22https://rpc.walletconnect.com/v1/?chainId=eip155:137&projectId=a28dc568ea31bc4224916d00dfcf0190%22,) code=SERVER_ERROR, version=web/5.7.1)

Call Stack
Logger.makeError
node_modules\@ethersproject\logger\lib.esm\index.js (224:0)
Logger.throwError
node_modules\@ethersproject\logger\lib.esm\index.js (233:0)
eval
node_modules\@ethersproject\web\lib.esm\index.js (231:0)
Generator.next
<anonymous>
fulfilled
node_modules\@ethersproject\web\lib.esm\index.js (5:42)

any one a idea?

**Comment by xzilja** [comment]
Looks like an issue with rpc call, perhaps missformated data. Do you have a reproduction repo for this I could look at / run locally to see what the error could be in more depth?
